### PR TITLE
Add menpo3d v0.8.0

### PR DESCRIPTION
**Pull request by patricksnape**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by patricksnape** [comment]
@conda-forge/staged-recipes Ready for review

File: recipes/menpo3d/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+     - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - cython
+    - numpy
```
Comment (beckermr) [inline recipes/menpo3d/meta.yaml]: Are you linking against numpy? You may need numpy in the run section per https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (patricksnape) [reply recipes/menpo3d/meta.yaml]: Yes apologies - added the `pin_compatible`

**Comment by patricksnape** [comment]
Thanks @beckermr !

**Pull request merged**